### ISSUE TITLE
BLAST-2.2.26 broken url

### ISSUE THREAD

**Issue by shahzebsiddiqui** [hpcugent/easybuild-easyconfigs#3727]
Hi Ken,

I notice the url for blast is not working. I tried to build it and it gives the following

```
[hpcswadm@hpcv18 BLAST]$ eb BLAST-2.2.26-Linux_x86_64.eb 
== temporary log file in case of crash /hpc/grid/scratch/tmp/eb-_b4R6o/easybuild-uw_ZWr.log
== processing EasyBuild easyconfig /hpc/hpcswadm/easybuild/BLAST/BLAST-2.2.26-Linux_x86_64.eb
== building and installing BLAST/2.2.26-Linux_x86_64...
== fetching files...
== FAILED: Installation ended unsuccessfully (build directory: /nfs/grid/software/RHEL7-BUILD/easybuild/build/BLAST/2.2.26/dummy-dummy-Linux_x86_64): build failed (first 300 chars): Couldn't find file blast-2.2.26-x64-linux.tar.gz anywhere, and downloading it didn't work either... Paths attempted (in order): /hpc/hpcswadm/easybuild/BLAST/b/BLAST/blast-2.2.26-x64-linux.tar.gz, /hpc/hpcswadm/easybuild/BLAST/BLAST/blast-2.2.26-x64-linux.tar.gz, /hpc/hpcswadm/easybuild/BLAST/blast-
== Results of the build can be found in the log file(s) /hpc/grid/scratch/tmp/eb-_b4R6o/easybuild-BLAST-2.2.26-20161024.104715.LpkJK.log
ERROR: Build of /hpc/hpcswadm/easybuild/BLAST/BLAST-2.2.26-Linux_x86_64.eb failed (err: "build failed (first 300 chars): Couldn't find file blast-2.2.26-x64-linux.tar.gz anywhere, and downloading it didn't work either... Paths attempted (in order): /hpc/hpcswadm/easybuild/BLAST/b/BLAST/blast-2.2.26-x64-linux.tar.gz, /hpc/hpcswadm/easybuild/BLAST/BLAST/blast-2.2.26-x64-linux.tar.gz, /hpc/hpcswadm/easybuild/BLAST/blast-")
```

I tried to update the url and I was able to proceed with install but I got an error, most likely this will need to be reworked. I have attached my updated file for version 2.2.26

The error I am getting is the following

```
[hpcswadm@hpcv18 BLAST]$ eb BLAST-2.2.26-Linux_x86_64.eb 
== temporary log file in case of crash /hpc/grid/scratch/tmp/eb-ytmJQz/easybuild-QLdRw9.log
== processing EasyBuild easyconfig /hpc/hpcswadm/easybuild/BLAST/BLAST-2.2.26-Linux_x86_64.eb
== building and installing BLAST/2.2.26-Linux_x86_64...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== building...
== testing...
== installing...
== taking care of extensions...
== postprocessing...
== sanity checking...
== FAILED: Installation ended unsuccessfully (build directory: /nfs/grid/software/RHEL7-BUILD/easybuild/build/BLAST/2.2.26/dummy-dummy-Linux_x86_64): build failed (first 300 chars): Sanity check failed: no (non-empty) directory of ('lib', 'lib64') in /nfs/grid/software/RHEL7/easybuild/software/BLAST/2.2.26-Linux_x86_64
== Results of the build can be found in the log file(s) /hpc/grid/scratch/tmp/eb-ytmJQz/easybuild-BLAST-2.2.26-20161024.105454.TRrzj.log
ERROR: Build of /hpc/hpcswadm/easybuild/BLAST/BLAST-2.2.26-Linux_x86_64.eb failed (err: "build failed (first 300 chars): Sanity check failed: no (non-empty) directory of ('lib', 'lib64') in /nfs/grid/software/RHEL7/easybuild/software/BLAST/2.2.26-Linux_x86_64")
[hpcswadm@hpcv18 BLAST]$ vi /hpc/grid/scratch/tmp/eb-ytmJQz/easybuild-BLAST-2.2.26-20161024.105454.TRrzj.log
```

If you can please fix this in the next release that would be great.

Thanks,

[BLAST-2.2.26-Linux_x86_64.txt](https://github.com/hpcugent/easybuild-easyconfigs/files/556722/BLAST-2.2.26-Linux_x86_64.txt)

**Comment by boegel** [comment]
@shahzebsiddiqui the sanity check error you are seeing happens because you commented out the `sanity_check_paths` block in the easyconfig file, which makes `eb` check for a non-empty `bin` and `lib`/`lib64` directory (the latter is not there for BLAST), see also http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#sanity-check

The download problem seems to be caused by a name mistake, since 2.2.26 is actually `BLAST+` rather than `BLAST`. @pescobar?

**Comment by boegel** [comment]
download URL fixed in #3861
